packages/appsync-modelgen-plugin/src/__tests__/visitors/__snapshots__/appsync-swift-visitor.test.ts.snap

```diff
@@ -0,0 +1,95 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AppSyncSwiftVisitor Should handle nullability of lists appropriately 1`] = `
+"// swiftlint:disable all
+import Amplify
+import Foundation
+
+public struct ListContainer: Model {
+  public let id: String
+  public var name: String?
+  public var list: [Int?]?
+  public var requiredList: [String?]
+  public var requiredListOfRequired: [StatusEnum]
+  public var listOfRequired: [Bool]?
+  public var requiredListOfRequiredDates: [Temporal.Date]
+  public var listOfRequiredFloats: [Double]?
+  public var requiredListOfCustomTypes: [CustomType?]
+  
+  public init(id: String = UUID().uuidString,
+      name: String? = nil,
+      list: [Int?]? = [],
+      requiredList: [String?] = [],
+      requiredListOfRequired: [StatusEnum] = [],
+      listOfRequired: [Bool]? = [],
+      requiredListOfRequiredDates: [Temporal.Date] = [],
+      listOfRequiredFloats: [Double]? = [],
+      requiredListOfCustomTypes: [CustomType?] = []) {
```

Comment:
For a listNullable field like `[var]` or `[var!]`, should it be initialized as `nil` instead?

Reply:
@lawmicha Do you think we should change these defaults to `nil` for optional lists?

Reply:
are there any default values for other platforms that we have to make a similar decision? Seems like the right thing to do is to set it to the most empty value, which would be nil

Reply:
but in doing so, we should re-evaluate what that looks like for developers in terms of the upgrade (when the regenerate the files, do they have any code changes to do?). before, they may have been able to do something like we just need to communicate to the developers the change to default values in-addition to the change in nullability of each type
```
let listContainer = ListContainer() // default creates empty lists
listContainer.listofRequiredFloats?.append(1.0) // directly append, without needing to instantiate it
```

Reply:
Java does not support default values for method parameters but all the list variables (optional or required) will be `null` if not initialized. In typescript they will be `undefined` if not initialized. I agree that setting the optionals to `nil` by default makes sense for Swift.

Reply:
Changed the defaults to `nil` for optional types.